app/src/main/AndroidManifest.xml

```diff
@@ -16,12 +17,29 @@
             </intent-filter>
         </activity>
         <activity
+
+            android:name="swen_anigains.mathematicfanatic.ClassroomActivity"
+            android:label="@string/title_activity_classroom"
+            android:parentActivityName=".MainActivity"
```

Comment:
The parent of the classroom is the main activity? This should probably be the locker.